samples/azure/blob-storage-databus-cleanup-function/ABSDataBus_4/DataBusBlobCleanupFunctions/DataBusCleanupOrchestrator.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+
+public static class DataBusCleanupOrchestrator
+{
+    #region DataBusCleanupOrchestratorFunction
+
+    [FunctionName(nameof(DataBusCleanupOrchestrator))]
+    public static async Task RunOrchestrator([OrchestrationTrigger] IDurableOrchestrationContext context, ILogger log)
+    {
+        var blobData = context.GetInput<DataBusBlobData>();
+
+        log.LogInformation($"Orchestrating deletion for blob at {blobData.Path} with ValidUntilUtc of {blobData.ValidUntilUtc}");
+
+        var validUntilUtc = DataBusBlobTimeoutCalculator.ToUtcDateTime(blobData.ValidUntilUtc);
+
+        DateTime timeoutUntil;
+
+        //Timeouts currently have a 7 day limit, use 6 day loops until the wait is less than 6 days
+        do
+        {
+            timeoutUntil = validUntilUtc > context.CurrentUtcDateTime.AddDays(2) ? context.CurrentUtcDateTime.AddDays(2) : validUntilUtc;
```

Comment:
It says 6 days loops here but we are using `AddDays(2)` to calculate the loop time. Shouldn't this then been using 

```
timeoutUntil = validUntilUtc > context.CurrentUtcDateTime.AddDays(6) ? context.CurrentUtcDateTime.AddDays(6) : validUntilUtc;
```

?